packages/domains-table/src/domains-table/domains-table-row.tsx

```diff
@@ -197,6 +200,17 @@ export function DomainsTableRow( { domain }: DomainsTableRowProps ) {
 					);
 				}
 
+				if ( column.name === 'ssl' && config.isEnabled( 'hosting-overview-refinements' ) ) {
```

Comment:
```suggestion
				if ( column.name === 'ssl' ) {
```
Not strictly necessary, but since we already check the `hosting-overview-refinements` config flag in `packages/domains-table/src/domains-table-header/columns.ts`, I think we might as well drop it here. Especially since we throw an error on L239 if a column doesn't map to any component.